src/core/default/commands/function/execute.py

```diff
@@ -38,46 +34,48 @@ def add_arguments(self, parser: ArgumentParser) -> None:
 
     def command(self, *args, **kwargs) -> None:
 
-        full_function_name: str = kwargs.get("function_id")
+        event_data = self._get_event_data(*args, **kwargs)
+
+        full_function_name = kwargs.get("function_id")
+        component_name, function_name = self.get_component_and_resource_from_qualified_name(full_function_name)
+
+        cloud_name = get_cloud_id_from_cdev_name(component_name, function_name)
+
+        lambda_client = client("lambda")
 
+        self.stdout.write(f"executing {full_function_name}")
+        response = lambda_client.invoke(
+            FunctionName=cloud_name,
+            InvocationType="RequestResponse",
+            Payload=json.dumps(event_data),
+        )
+
+        print(response)
```

Comment:
Looking at it now, I had made a mistake using `print` here. Commands should send all output to `self.stdout`.